### DISCUSSION TITLE
v3版本官网文档过时

### DISCUSSION THREAD

**Discussion by victor-vb** [umijs/umi#10926] [Show and tell]
当官网和实际产品的说明文档都不一致。那这个产品还是闭源的好，使用v3x版本按照官方列子跑，竟然框架骨骼目录结构都不一致。不知道要踩多少坑

举个官方列子
![image](https://user-images.githubusercontent.com/113876742/230755885-d07215f9-8c25-4715-9bd4-0ade808092aa.png)
![image](https://user-images.githubusercontent.com/113876742/230755902-74c62e3b-5e07-4dcc-9cc2-f8234c9f34f0.png)
v3脚手架初始化项目，.env文件无，手动创建。输入环境变量，实际上并未生效。
![image](https://user-images.githubusercontent.com/113876742/230755936-a54fd7b4-8ab6-4005-a6e2-c557a3be095a.png)
必须得这样才生效。文档是真的有点拉跨

**Comment by DearGaz** [comment]
同感，乖乖按引导升级，结果一堆错误。。。。升级的心累

**Reply by victor-vb** [reply]
玩ant.design看到又用这这个框架，本来想拿来跑跑，学习一下，学习的价值不大，哎。工具本来是简化工作量的，结果一翻文档，对不上就新很累

**Comment by fz6m** [comment]
umi 3 不维护了，建议升级到 umi 4
